tools/update_vendor.sh

```diff
@@ -1,13 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
is 2018 the correct date for this file?

Reply:
Yes it is. The file is initially been added in 2018.